JsonSchema.Generation/PropertyNameResolvingMethods.cs

```diff
@@ -0,0 +1,28 @@
+using System.Linq;
+using System.Reflection;
+using System.Text.Json.Serialization;
+
+namespace Json.Schema.Generation;
+
+/// <summary>
+/// Declares a property name resolution which is used to provide a property name.
+/// </summary>
+/// <param name="input">The property.</param>
+/// <returns>The property name or null, if no name was resolved.</returns>
+public delegate string? PropertyNameResolvingMethod(MemberInfo input);
```

Comment:
Please rename to `PropertyNameResolver` (and the config property as well).

Reply:
done